### DISCUSSION TITLE
Request：延べアクセス回数をカウントするページアクセスカウンタが欲しい

### DISCUSSION THREAD

**Discussion by hide-o-mvd** [growilabs/growi#7716] [Ideas / Requests]
Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        | CentOS 7.8.2003|
|GROWI     |4.1.11|
|node.js   |14.15.0|
|npm       |6.14.8|
|Using Docker|yes|
|Using [growi-docker-compose][growi-docker-compose]|yes|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*

### Client

| item     | version |
| ---      | --- |
|OS        | Windows10 |
|browser   | Chrome86.0.4240.193 |

How to reproduce? (再現手順)
---------------------------

1. 任意のページにアクセスする

What is the expected result? (期待される動作)
-------------------------------------------

1. 延べアクセス数をAPI経由で入手できる

Note
----
日々活用させていただいております。ありがとうございます。

現状、アクセスカウンタに類するものはいいね数（Liker）とユニークユーザー情報（seenUsers）があるかと思いますが、これに加えてページ毎の延べアクセス数のカウンタがあるとありがたいです。

「１度アクセスされたら終わり」と「何度もアクセスされている」ページの違いが現状のカウンタ情報からは判断できず、延べカウンタがあれば後者のほうが価値が高いページとして取り上げることができると考えています。
※いいねは押す基準がユーザーごとに異なるため、この判断材料として使うことは難しいです。

カウンタ値の情報としてはページ内に表示されなくても、API経由で入手できれば（私としては）問題ありません。

**Comment by itizawa** [comment]
機能をリクエストしていただきありがとうございます。

すぐに実装とはいきませんがチーム内で検討させていただきます。

**Comment by shield-9** [comment]
再訪などの分析が目的の場合、アクセス数だけだと単純にアクセスする人数が多いのか区別がつかないので、目的を達成するにはGoogle Analyticsなどを埋め込んだ方が早いかもしれないですね。
